docs/appendix/glossary.md

```diff
@@ -22,6 +22,10 @@ Placeholder
 
 Placeholder
 
+### Wallet
+
+Placeholder
+
```

Comment:
glossary is useful, any plans to update the placeholders?

Reply:
not in this MR but yeah